.gitignore

```diff
@@ -121,3 +121,4 @@ common/autoinstallers/*/.npmrc
 # Disallow other package manager lock-file
 /package-lock.json
 /yarn.lock
+quick-lint-js.config
```

Comment:
what is this

Reply:
config file of quick-lint-js, a fast js lint tool